### DISCUSSION TITLE
Severity "Success" present in Hl7.Fhir.R4 package but was not introduced until in R5 of the FHIR specification?

### DISCUSSION THREAD

**Discussion by GabrielOrtizElypta** [FirelyTeam/firely-net-sdk#2974] [Q&A] [answered]
Hi,  I am using the C# package Hl7.Fhir.R4 and in its source code, severity has the value Success which is not defined in R4 according to https://hl7.org/fhir/R4/valueset-issue-severity.html
but was added in R5 according to https://hl7.org/fhir/codesystem-issue-severity.html

Is this an error in the Hl7.Fhir.R4 package or am I reading the specification incorrectly?

**Comment by ewoutkramer** [comment, marked as answer]
There is a limited amount of classes (of which OperationOutcome (which uses issue-severity) is one) that are in our Base/Conformance libraries, which means that they are shared across FHIR versions. These classes are generally speaking generated according to the latest version of FHIR, so in this case it means that Success is there, even though it is not allowed in R4.  We decided to design it this way since the advantage of having these classes shared far outweighs the fact that you can select the incorrect code (just don't do it ;-).

**Reply by GabrielOrtizElypta** [reply]
Great, thank you so much for the clarification. I'll keep it in mind when using classes from those libraries.
